### ISSUE TITLE
Outputs twice

### ISSUE THREAD

**Issue by SephVelut** [mikeerickson/gulp-phpunit#57]
upgrading to 0.19 from 0.18 makes the test result output happen twice

```
gulp.task('phpunit', function() {
	var options = {debug: false, notify: false};
	gulp.src('phpunit.xml').pipe(phpunit('./vendor/bin/phpunit', options));
});
```

**Comment by mikeerickson** [comment]
@sephvelut Thanks for report, I will have a look at this shortly.  Seems there was regression issue when 0.19 update was released (I have addressed this issue at some point many releases ago).

**Comment by mikeerickson** [comment]
Issue has been fixed with v0.20.0 release

https://www.npmjs.com/package/gulp-phpunit
